DESCRIPTION

```diff
@@ -18,7 +18,7 @@ Description: Tools for processing single cell data associated with the
 License: BSD_3_clause + file LICENSE
 Encoding: UTF-8
 LazyData: true
-RoxygenNote: 7.2.0
+RoxygenNote: 7.2.1
```

Comment:
Josh and I have been going back and forth on committing and removing this change because for some reason we have different versions but can't figure out why.

Reply:
Happy to join in the party!